integreat_cms/cms/views/pois/poi_list_view.py

```diff
@@ -124,6 +124,7 @@ def get(self, request, *args, **kwargs):
                 "language": language,
                 "languages": region.active_languages,
                 "search_query": query,
+                "source_language": region.get_source_language(language.slug),
```

Comment:
```suggestion
                "source_language": region.get_source_language(language.slug),
                "content_type": "locations",
```

Apply the same change to page tree view too.